exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/AggregateByLabelMetricTimeSeriesBuilder.java

```diff
@@ -55,10 +58,20 @@ public final class AggregateByLabelMetricTimeSeriesBuilder implements MetricTime
   private final Map<MetricWithLabels, TimeSeries.Builder> pendingTimeSeries = new HashMap<>();
   private final String projectId;
   private final String prefix;
+  private final Predicate<AttributeKey<?>> resourceAttributeFilter;
 
+  @Deprecated
   public AggregateByLabelMetricTimeSeriesBuilder(String projectId, String prefix) {
     this.projectId = projectId;
     this.prefix = prefix;
+    this.resourceAttributeFilter = MetricConfiguration.NO_RESOURCE_ATTRIBUTES;
+  }
+
+  public AggregateByLabelMetricTimeSeriesBuilder(
```

Comment:
Does this mean users are required to pass a predicate in when they construct an exporter?  Or is this function not really meant to be used directly by users?

Reply:
IMO, this function is not meant to be used directly by the customers. 

However, because it was already public, I added a second constructor instead of updating the old one to avoid breaking changes.

Reply:
Sounds good